src/Bathymetry.jl

```diff
@@ -228,7 +229,7 @@ function interpolate_bathymetry_in_passes(native_h, target_grid;
     h_data = Array(interior(target_h, :, :, 1))
 
     if minimum_depth > 0
-        shallow_ocean = h_data .> minimum_depth
+        shallow_ocean = h_data .> - minimum_depth
```

Comment:
```suggestion
        shallow_ocean = z_data .> - minimum_depth
```

`h` is depth, `z` is vertical coordinate...

Reply:
and you would have probably wanted `h < minimum_depth` right?

Reply:
In this case `h` is the topography as measured from sea-level, so positive for mountains and negative for the seabed. What about just `topography` as a name?

Reply:
Use the symbol "z" somehow to indicate that it's a z-coordinate

Reply:
`z_topography` if you like